gridcomps/Cap/MAPL_CapGridComp.F90

```diff
@@ -38,6 +38,12 @@ module MAPL_CapGridCompMod
 
   public :: MAPL_CapGridComp, MAPL_CapGridCompCreate, MAPL_CapGridComp_Wrapper
 
+  type :: Throughput_Timer
+     real(kind=real64) :: loop_start_timer
+     real(kind=REAL64) :: start_run_timer
+     real(kind=REAL64) :: start_timer
+  end type
+
```

Comment:
Since I'm asking for additional changes anyway, I'm going to request a style change here:

Use CamelCase for the type name: `ThroughputTimers`   (note the rare plural for a type)
I was also going to suggest new names for the components inside, but I did not really come up with anything I liked.    I suspect at some point we should just use  ordinary timers from profiler.  (Won't help with the names though.)

Reply:
changed